### ISSUE TITLE
Fleet "continous delivery dashboard" should show Clusters/Bundles/Resources in error at "mouse over"

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#10339]
Internal Reference: SURE-5804

In the fleet dashboard, when a Cluster,Bundle or Resource has errors it would be nice if the mouse over would show those which are in the error state.

**Comment by gaktive** [comment]
Talk to @kwwii if there are any questions but the design is locked in. The backend will return an error state so there's no reason to dig into the reason state.

**Comment by aalves08** [comment]
@edenhernandez-suse have all the issues been resolved here about the content and the states? Am I supposed to replace the current tooltip's content?

**Comment by edenhernandez-suse** [comment]
@gaktive @kwwii it currently shows how many items are in different states:

<img width="287" alt="image" src="https://github.com/rancher/dashboard/assets/120640736/0d1e1c12-38e6-4243-b613-ed6859cefceb">

If we change that for a simple list with all the items with just any of those states, we might be losing information.

But even if we consider that solution as a good option, it looks like what we get is just the amount of items per state, so I believe @aalves08 can't infer which items are in those error states. That's why we need to talk with someone in the backend team, but I don't know which team is in charge of Fleet.

**Comment by aalves08** [comment]
We are getting, per item, the statuses for each of the `bundles` and `resources`, not for clusters. That shouldn't be a problem, I think, since we need to get all of the clusters belonging to that particular namespace.

The main question that @edenhernandez-suse had (correct me if I am wrong) is what constitutes an "error state"... Being "not ready" is an error?

Also, are we sure we want to replace the content of the current tooltip (as shown in comment before this one) with the new content?

**Comment by aalves08** [comment]
@nwmac according to our conversation, should we bump this to a different milestone?

**Comment by nwmac** [comment]
@aalves08 Is there much left to do on this? Should we push this to 2.10 to give ourselves time?

**Comment by aalves08** [comment]
@nwmac tbh, I don't even remember what was the last solution me and @edenhernandez-suse agreed upon, but I remember I chatted with you about some issue and you told me to put it to a halt. I would say we should revisit this in `2.10`

**Comment by edenhernandez-suse** [comment]
@aalves08 didn't you fix the Cluster count and finish the badges to make them clickable? You sent me a video with the working solution and it was perfect.

**Comment by edenhernandez-suse** [comment]
Yes, the new layout makes this ticket outdated. There might be other issues, but this specific one does not apply anymore.
